README.md

```diff
@@ -1,31 +1,9 @@
 # Lego3
```

Comment:
Where is all the text?

Reply:
In https://github.com/Elyash/Lego3-WorkStation
The PR will open soon